### DISCUSSION TITLE
Breezy widgets tap actions

### DISCUSSION THREAD

**Discussion by whitewallman** [breezy-weather/breezy-weather#1443] [Q&A] [answered]
Is it possible to disable Breezy widgets tap actions? I constantly accidentally tap the widget and open the clock or weather.

**Comment by papjul** [comment]
How do you "accidentally" tap the widget?

**Reply by whitewallman** [reply]
Touching the screen inadvertently, tap instead of swipe, etc..
Or if one prefers not to have a tap action for whatever reason.

**Reply by papjul** [reply, marked as answer]
From what I read, it doesn't look like an issue specific to Breezy Wearher widgets.

If I understand correctly:
- it can happen on any part of the homescreen, not just Breezy Weather widget, so sometimes you inadvertently tap another app icon and open it when you didn't want, right? Otherwise, if it only affects a specific part of the screen (for example, you only have this issue on the lower part of the screen because that's where your fingers lay), then I will suggest you move the widget to an area of the screen where you know you won't inadvertently tap
- it can happen inside an app? Sometimes you inadvertently tap an action or confuse swipe and tap and accidently launch an action you didn't want? How do you usually handle this with other apps? I mean it looks like a much broader issue than just Breezy Weather widgets

Also, have a look at Android settings to adjust sensitivity of the touchscreen. I know sometimes there is a setting enabled to use the touchscreen with gloves that make the sensivity higher. Disabling it might help

**Reply by whitewallman** [reply]
Thanks. I was actually just looking for a "Disable Tap Actions" toggle in the Breezy settings/widgets and live wallpaper section. In other words, I'd prefer the widget to be visual only (clock/weather). Not launch clock settings or weather app if that makes sense.

**Reply by papjul** [reply]
I understand, but this option doesn't exist, so please try the alternatives suggested ;)

**Reply by whitewallman** [reply]
Gotcha. Thanks!
